mwdb/model/file.py

```diff
@@ -113,7 +174,7 @@ def open(self):
                 app_config.mwdb.s3_storage_region_name,
                 app_config.mwdb.s3_storage_secure,
             ).get_object(app_config.mwdb.s3_storage_bucket_name, self._calculate_path())
-        else:
+        elif app_config.mwdb.storage_provider == StorageProviderType.DISK:
```

Comment:
Maybe add an `else` that raises and exception? I know it would be technically unreachable, however it would be a sanity check it someone adds a new storage provider

Reply:
Agree, possible regression bug here. Todo :+1:

Reply:
Done :+1: